var/spack/repos/builtin/packages/tau/package.py

```diff
@@ -93,7 +93,7 @@ class Tau(Package):
     depends_on('likwid', when='+likwid')
     depends_on('papi', when='+papi')
     depends_on('libdwarf', when='+libdwarf')
-    depends_on('libelf', when='+libdwarf')
+    depends_on('elf', when='+libdwarf')
```

Comment:
Shouldn't this be `when='+elf'`?

Reply:
Yes... I need to amend this. I tested it with those changes in place but I think I had a copy-paste error dropping it into the pull request.